### ISSUE TITLE
the trait bound `Chess: Syzygy` is not satisfied

### ISSUE THREAD

**Issue by lzanini** [niklasf/shakmaty-syzygy#19]
Hi, 

When I try to use the crate like this

```rust
use shakmaty::Chess;
use shakmaty_syzygy::{Tablebase};

struct SyzygyData {
    tables: Tablebase<Chess>,
}
```

I get the following errors:
```
the trait bound `Chess: Syzygy` is not satisfied
the trait `Syzygy` is implemented for `shakmaty::position::Chess`
```

```
the trait bound `Chess: shakmaty::position::Position` is not satisfied
the trait `shakmaty::position::Position` is implemented for `shakmaty::position::Chess`
```

Which seems odd, as shakmaty::position is private. 

I am using the latest versions of shakmaty and shakmaty-syzygy.

Any help would be appreciated !

**Comment by niklasf** [comment]
Hi, This kind of error would be typical for a situation where the `shakmaty` version in your own crate does not match the one used by `shakmaty-syzygy`. However, if you were using the latest version of both, that should not be the case.

Incidentally, I just released updates for both crates (`shakmaty = 0.24`, `shakmaty-syzygy = 0.22`). Can you please try those?

If the error persists, please share your `Cargo.lock` file.

**Comment by lzanini** [comment]
Updating to the latest version fixed it. Thanks !
